src/main/kotlin/no/nav/familie/ef/mottak/config/ApplicationConfig.kt

```diff
@@ -102,7 +103,14 @@ class ApplicationConfig {
     }
 
     @Bean
-    fun kotlinModule(): KotlinModule = KotlinModule()
+    fun kotlinModule(): KotlinModule = KotlinModule.Builder()
```

Comment:
Her hadde det ju vært fint med en kommentar i PR koden som forklarte hvorfor disse settes slik. 
Ser dette var default for `KotlinModule()`, og er også default for Buildern, så trenger vi egentlige å sette de?

Reply:
Vi trenger nok ikke sette dem, men intellij lagde builderen sånn for meg, fordi det er det replaceWith-delen av deprected-annotasjonen foreslår.

Reply:
Godt observert.